Server/ConcordiaCurriculumManager/Repositories/DossierRepository.cs

```diff
@@ -37,10 +38,12 @@ public async Task<bool> SaveCourseCreationDossier(CourseCreationDossier dossier)
             
     }
 
-    public async Task<bool> SaveDossier(Dossier dossier) {
-        await _dbContext.Dossiers.AddAsync(dossier);
-        var result = await _dbContext.SaveChangesAsync();
-        return result > 0;
-    }
-
+    public async Task<Dossier?> GetDossierByDossierId(Guid dossierId) => await _dbContext.Dossiers.Where(d => d.Id == dossierId).FirstOrDefaultAsync();

```

Comment:
Does this return the associated CourseCreationRequests, or are they null? Also, can you write a test for this method?